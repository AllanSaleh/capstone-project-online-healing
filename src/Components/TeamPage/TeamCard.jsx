import React from 'react';

export default function TeamCard({ img, name, position }) {
  return (
    <div className="flex flex-col w-full my-4 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2">
      <div className="lg:h-60">
        <img className="w-full" src={img} alt="MemberImage" />
        <div className="h-2 w-full bg-green-300" />
      </div>
      <div className="h-48 flex flex-col justify-around items-center">
        <div className="text-gray-700 text-3xl lg:text-subtitle text-center">{name}</div>
        <div className="text-gray-700 text-lg lg:text-paragraph text-center w-40 lg:w-auto">{position}</div>
      </div>
    </div>
  );
}
