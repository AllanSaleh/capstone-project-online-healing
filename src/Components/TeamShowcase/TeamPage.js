import React from 'react';

import NavBar from '../NavBar/NavBar';
import Footer from '../Footer/Footer';

import AllanImg from './AllanImg.svg';
import PayamImg from './PayamImg.svg';
import AhmedImg from './AhmedImg.svg';
import AbdulmajeedImg from './AbdulmajeedImg.svg';
import SnoorImg from './SnoorImg.svg';
import AvrazImg from './AvrazImg.svg';

export default function Recomended() {
  const Team = [
    {
      img: AllanImg,
      name: 'Allan Saleh',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: PayamImg,
      name: 'Payam Abubakr',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: AhmedImg,
      name: 'Ahmed Mohammed',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: AbdulmajeedImg,
      name: 'Abdulmajeed Jaafer',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: SnoorImg,
      name: 'Snoor Madih',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: AvrazImg,
      name: 'Avraz Zebary',
      position: 'Lead Engineer & Web Developer',
    },
  ];
  return (
    <div>
      <NavBar />
      <div className="flex flex-col justify-evenly px-sides pt-navbar h-firstsection">
        <div className="text-title uppercase">we are healing, nice to meet you!</div>

        <div className="flex flex-col justify-between">
          <div className="text-subtitle text-gray-700 mb-8">Meet the Team!</div>

          <div className="flex justify-between">
            {Team.map((member) => (
              <div className="flex flex-col w-52 bg-blue-light rounded-3xl justify-between">
                <div className="h-60">
                  <img className="w-full" src={member.img} alt="MemberImage" />
                  <div className="h-2 w-full bg-green-300" />
                </div>
                <div className="h-44 flex flex-col justify-evenly">
                  <div className="text-gray-700 text-subtitle text-center">{member.name}</div>
                  <div className="text-gray-700 text-paragraph text-center">{member.position}</div>
                </div>
              </div>
            ))}
          </div>
        </div>
      </div>
      <Footer />
    </div>
  );
}
