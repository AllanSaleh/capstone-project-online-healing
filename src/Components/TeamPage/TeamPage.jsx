import React from 'react';

import TeamCard from './TeamCard';

import AllanImg from './Images/AllanImg.svg';
import PayamImg from './Images/PayamImg.svg';
import AhmedImg from './Images/AhmedImg.svg';
import AbdulmajeedImg from './Images/AbdulmajeedImg.svg';
import SnoorImg from './Images/SnoorImg.svg';
import AvrazImg from './Images/AvrazImg.svg';

export default function TeamPage() {
  window.scrollTo(0, 0);
  const Team = [
    {
      img: AllanImg,
      name: 'Allan Saleh',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: PayamImg,
      name: 'Payam Abubakr',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: AhmedImg,
      name: 'Ahmed Mohammed',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: AbdulmajeedImg,
      name: 'Abdulmajeed Jaafer',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: SnoorImg,
      name: 'Snoor Madih',
      position: 'Lead Engineer & Web Developer',
    },
    {
      img: AvrazImg,
      name: 'Avraz Zebary',
      position: 'Lead Engineer & Web Developer',
    },
  ];
  return (
    <div className="flex flex-col justify-evenly px-sides pt-navbar lg:h-firstsection">
      <div className="text-3xl lg:text-title uppercase">we are healing, nice to meet you!</div>

      <div className="flex flex-col justify-between">
        <div className="text-lg lg:text-subtitle text-gray-700 mb-8">Meet the Team!</div>

        <div className="flex justify-between flex-col lg:flex-row">
          {Team.map((member) => (
            <TeamCard img={member.img} name={member.name} position={member.position} />
          ))}
        </div>
      </div>
    </div>
  );
}
