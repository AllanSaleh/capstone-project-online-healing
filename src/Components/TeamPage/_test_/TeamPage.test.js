import * as React from 'react';
import TeamPage from "../TeamPage";
import * as renderer from "react-test-renderer";

it('Team Page renders correctly', () => {
  const tree = renderer
    .create(<TeamPage/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});