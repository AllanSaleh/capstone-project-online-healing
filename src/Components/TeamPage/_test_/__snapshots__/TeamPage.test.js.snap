// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Team Page renders correctly 1`] = `
<div
  className="flex flex-col justify-evenly px-sides pt-navbar lg:h-firstsection"
>
  <div
    className="text-3xl lg:text-title uppercase"
  >
    TeamPage.Title
  </div>
  <div
    className="flex flex-col justify-between"
  >
    <div
      className="text-lg lg:text-subtitle text-gray-700 mb-8"
    >
      TeamPage.Subtitle
    </div>
    <div
      className="flex justify-between flex-col lg:flex-row"
    >
      <div
        className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2"
      >
        <div
          className="lg:h-60"
        >
          <img
            alt="MemberImage"
            className="lg:w-full"
            src="AllanImg.svg"
          />
          <div
            className="h-2 w-full bg-green-300"
          />
        </div>
        <div
          className="h-48 flex flex-col justify-around"
        >
          <div
            className="text-gray-700 text-lg lg:text-subtitle text-center"
          >
            Allan Saleh
          </div>
          <div
            className="text-gray-700 text-lg lg:text-paragraph text-center"
          >
            Lead Engineer & Web Developer
          </div>
        </div>
      </div>
      <div
        className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2"
      >
        <div
          className="lg:h-60"
        >
          <img
            alt="MemberImage"
            className="lg:w-full"
            src="PayamImg.svg"
          />
          <div
            className="h-2 w-full bg-green-300"
          />
        </div>
        <div
          className="h-48 flex flex-col justify-around"
        >
          <div
            className="text-gray-700 text-lg lg:text-subtitle text-center"
          >
            Payam Abubakr
          </div>
          <div
            className="text-gray-700 text-lg lg:text-paragraph text-center"
          >
            Lead Engineer & Web Developer
          </div>
        </div>
      </div>
      <div
        className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2"
      >
        <div
          className="lg:h-60"
        >
          <img
            alt="MemberImage"
            className="lg:w-full"
            src="AhmedImg.svg"
          />
          <div
            className="h-2 w-full bg-green-300"
          />
        </div>
        <div
          className="h-48 flex flex-col justify-around"
        >
          <div
            className="text-gray-700 text-lg lg:text-subtitle text-center"
          >
            Ahmed Mohammed
          </div>
          <div
            className="text-gray-700 text-lg lg:text-paragraph text-center"
          >
            Lead Engineer & Web Developer
          </div>
        </div>
      </div>
      <div
        className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2"
      >
        <div
          className="lg:h-60"
        >
          <img
            alt="MemberImage"
            className="lg:w-full"
            src="AbdulmajeedImg.svg"
          />
          <div
            className="h-2 w-full bg-green-300"
          />
        </div>
        <div
          className="h-48 flex flex-col justify-around"
        >
          <div
            className="text-gray-700 text-lg lg:text-subtitle text-center"
          >
            Abdulmajeed Jaafer
          </div>
          <div
            className="text-gray-700 text-lg lg:text-paragraph text-center"
          >
            Lead Engineer & Web Developer
          </div>
        </div>
      </div>
      <div
        className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2"
      >
        <div
          className="lg:h-60"
        >
          <img
            alt="MemberImage"
            className="lg:w-full"
            src="SnoorImg.svg"
          />
          <div
            className="h-2 w-full bg-green-300"
          />
        </div>
        <div
          className="h-48 flex flex-col justify-around"
        >
          <div
            className="text-gray-700 text-lg lg:text-subtitle text-center"
          >
            Snoor Madih
          </div>
          <div
            className="text-gray-700 text-lg lg:text-paragraph text-center"
          >
            Lead Engineer & Web Developer
          </div>
        </div>
      </div>
      <div
        className="flex flex-col w-40 mt-2 lg:w-52 bg-blue-light rounded-3xl justify-between mx-2"
      >
        <div
          className="lg:h-60"
        >
          <img
            alt="MemberImage"
            className="lg:w-full"
            src="AvrazImg.svg"
          />
          <div
            className="h-2 w-full bg-green-300"
          />
        </div>
        <div
          className="h-48 flex flex-col justify-around"
        >
          <div
            className="text-gray-700 text-lg lg:text-subtitle text-center"
          >
            Avraz Zebary
          </div>
          <div
            className="text-gray-700 text-lg lg:text-paragraph text-center"
          >
            Lead Engineer & Web Developer
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
